docs/integrations/delta-lake-pandas.md

```diff
@@ -0,0 +1,279 @@
+    .agg({"v1": "sum"})
+)
+```
+
+This query runs in 8 seconds, which is a significant performance enhancement.
+
+Now let's Z Order the Delta table by `id1` which will make the data skipping even better.  Run the query again on the Z Ordered Delta table:
+
+```python
+(
+    DeltaTable(f"{Path.home()}/data/deltalake_baseline_G1_1e9_1e2_0_0", version=1)
+    .to_pandas(filters=[("id1", "==", "id016")], columns=["id1", "id2", "v1"])
+    .query("id1 == 'id016'")
+    .groupby("id2")
+    .agg({"v1": "sum"})
+)
+```
+
+The query now executes in 2.4 seconds.
+
+Delta tables can make certain pandas queries run much faster.
+
+## Delta Lake lets pandas users time travel
+
+Start by creating a Delta table:
+
+```python
+from deltalake import write_deltalake, DeltaTable
+
+df = pd.DataFrame({"num": [1, 2, 3], "letter": ["a", "b", "c"]})
+write_deltalake("tmp/some-table", df)
+```
+
+Here are the contents of the Delta table (version 0 of the Delta table):
+
+```
++-------+----------+
+|   num | letter   |
+|-------+----------|
+|     1 | a        |
+|     2 | b        |
+|     3 | c        |
++-------+----------+
+```
+
+Now append two rows to the Delta table:
+
+```python
+df = pd.DataFrame({"num": [8, 9], "letter": ["dd", "ee"]})
+write_deltalake("tmp/some-table", df, mode="append")
+```
+
+Here are the contents after the append operation (version 1 of the Delta table):
+
+```
++-------+----------+
+|   num | letter   |
+|-------+----------|
+|     1 | a        |
+|     2 | b        |
+|     3 | c        |
+|     8 | dd       |
+|     9 | ee       |
++-------+----------+
+```
+
+Now perform an overwrite transaction:
+
+```python
+df = pd.DataFrame({"num": [11, 22], "letter": ["aa", "bb"]})
+write_deltalake("tmp/some-table", df, mode="overwrite")
+```
+
+Here are the contents after the overwrite operation (version 2 of the Delta table):
+
+```
++-------+----------+
+|   num | letter   |
+|-------+----------|
+|     8 | dd       |
+|     9 | ee       |
++-------+----------+
+```
+
+Read in the Delta table and it will grab the latest version by default:
+
+```
+DeltaTable("tmp/some-table").to_pandas()
+
++-------+----------+
+|   num | letter   |
+|-------+----------|
+|    11 | aa       |
+|    22 | bb       |
++-------+----------+
+```
+
+You can easily time travel back to version 0 of the Delta table:
+
+```
+DeltaTable("tmp/some-table", version=0).to_pandas()
+
++-------+----------+
+|   num | letter   |
+|-------+----------|
+|     1 | a        |
+|     2 | b        |
+|     3 | c        |
++-------+----------+
+```
+
+You can also time travel to version 1 of the Delta table:
+
+```
+DeltaTable("tmp/some-table", version=1).to_pandas()
+
++-------+----------+
+|   num | letter   |
+|-------+----------|
+|     1 | a        |
+|     2 | b        |
+|     3 | c        |
+|     8 | dd       |
+|     9 | ee       |
++-------+----------+
+```
+
+Time travel is a powerful feature that pandas users cannot access with CSV or Parquet.
+
+## Schema enforcement
```

Comment:
Do we perhaps want to have a short section on how you can overwrite the schema and the table at same time.

Reply:
@ion-elgreco - I actually just tried to do `write_deltalake("tmp/some-table", df, mode="overwrite")` thinking that would overwrite the scheme and table at the same time and it still gave a `ValueError` surprisingly.  Is there another syntax?  Should I also create an issue to add schema evolution?

Reply:
You need to also pass overwrite_schema = True